### PR TITLE
F [#PS-56] introduce legal_entity.business_tax_id_provided attribute

### PR DESCRIPTION
**Pull request by sadtuna**

### CODE REVIEW

File: lib/payment/response_classes/merchant_stripe.rb [hunk 1]
```diff
@@ -57,8 +57,8 @@ class LegalEntity < StripeHash
     include DobConvertible
 
     attr_accessor :additional_owners, :address, :business_name,
-                  :business_tax_id, :dob, :first_name, :last_name, :type,
-                  :verification
+                  :business_tax_id, :business_tax_id_provided, :dob,
+                  :first_name, :last_name, :type, :verification
```
Comment (ceigel) [inline lib/payment/response_classes/merchant_stripe.rb]: How is business_tax_id_provided different than business_tax_id?

Comment (sadtuna) [inline lib/payment/response_classes/merchant_stripe.rb]: business_tax_id_provided is just a flag, say if we previously provided the business_tax_id.

business_tax_id is a little different, we can set it, but never get back the value (only the business_tax_id_provided flag)

**Comment by cs** [comment]
Looks good to me! :+1:

@ceigel ?

**Comment by ceigel** [comment]
Looks good to me, too.
We should consider updating the contract, though.

**Comment by sadtuna** [comment]
I'll do the upgrading of the contract in my other pull request: https://github.com/shore-gmbh/payment-service/pull/115

**Pull request merged**